pkgs/by-name/co/config-store/package.nix

```
@@ -0,0 +1,30 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "config-store";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "DOD-101";
+    repo = pname;
```

```suggestion
    repo = "config-store";
```
See https://github.com/NixOS/nixpkgs/issues/277994